### ISSUE TITLE
Viewing whole thread by default.

### ISSUE THREAD

**Issue by Ep1cM1n10n** [ilosuna/mylittleforum#440]
I was looking and cant find anything to make me view the whole thread by default, as it only views the tree. i was hoping there could be an option added so every user by default when clicking on a thread opens the entire thing.

**Comment by auge8472** [comment]
@loesler I found the settings `board_view`, `mix_view` and `thread_view` in the advanced settings of the project forum. These settings does not exist in fresh installations of a current version (i.e. not in my testing forum, installed as 2.4.6). These settings does also not exist in [the wiki page for the forum configuration](https://github.com/ilosuna/mylittleforum/wiki/Configuration). If I'm right, these settings was a heritage of MLF1 and steered the default main view. Do you have any idea, when this was gone?

To come back to the issue of @Ep1cM1n10n, I found no setting for the default thread view, wether in the forum nor in the user settings. I would expect this to be at least a user setting.

**Comment by loesler** [comment]
> these settings was a heritage of MLF1

I don't know anything about this option - I'm sorry.

**Comment by jedomed** [comment]
Hi, have there been any updates to this? I'm also curious if there's a way to show full threads by default

**Comment by auge8472** [comment]
> Hi, have there been any updates to this? I'm also curious if there's a way to show full threads by default

No, unfortunately there is currently no progress on this feature request.
